ktor-server/jvm/src/io/ktor/server/application/plugins/api/ApplicationConfigExtensions.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.application.plugins.api
+
+import io.ktor.server.config.*
+
+/**
```

Comment:
Represents the port the current application is running on, as defined by the configuration at start-up.